pkg/agent/storage/backcompat.go

```diff
@@ -0,0 +1,60 @@
+package storage
+
+import (
+	"crypto/x509"
+	"path/filepath"
+)
+
+func Backcompat(dir string) (Storage, error) {
+	j, err := JSONFile(filepath.Join(dir, "data.json"))
```

Comment:
data.json feels kind of generic if we decide to have other files storing different agent data in the future. Could we have a name that would be more descriptive?